ios/Plugin/Plugin.swift

```diff
@@ -754,7 +771,16 @@ public class RadarPlugin: CAPPlugin, RadarDelegate {
 
                 return
             }
-            let units: RadarRouteUnits = unitsStr == "METRIC" || unitsStr == "metric" ? .metric : .imperial;
+            var units: RadarRouteUnits = .metric;
```

Comment:
We should probably bring this to Android to keep it consistent.

Reply:
I changed this because I found a bug in `getMatrix()` units parsing (https://github.com/radarlabs/capacitor-radar/blob/master/ios/Plugin/Plugin.swift#L856 should be `unitsStr.lowercased()`) and I figured it was best to share logic between the two functions on iOS. Agree we should probably copy to Android and squint at the implementations across all x-platform libs at some point. Historically I tried to keep these line-for-line similar across all libs and all platforms, but things have gotten fragmented recently